src/graphql.rs

```diff
@@ -729,6 +669,51 @@ fn get_trend(
     signal::filter::filtfilt(&b, &a, &original)
 }
 
+/// Checks after, before, first, and last to see if they follow GraphQL guidelines.
+///
+/// # Errors
+///
+/// Returns an error if the parameters are invalid.
+#[allow(clippy::type_complexity)]
+pub fn validate_pagination_params_and_set_default(
```

Comment:
No need to be pub since it doesn't need to be used outside of the crate it seems! 
I think validate_and_process_pagination_params or is good ,so I'll change it to that seems it seems to be most fitting.

Reply:
Done